test/unit/spec/MfaVerify_spec.js

```diff
@@ -214,8 +212,23 @@ function (Okta,
       return setup(res);
     }
 
+    var getAutoPushResponse = function (response, factorType, provider, autoPushVal) {
+      var responseCopy = Util.deepCopy(response);
+      var factors = responseCopy['response']['_embedded']['factors'];
+      var factor = _.findWhere(factors, {factorType: factorType, provider: provider});
+      var id = factor.id;
+      var policy = responseCopy['response']['_embedded']['policy'];
+
+      policy['factorsPolicyInfo'][id]['autoPushEnabled'] = autoPushVal;
+      return responseCopy;
+    };
+
+    var autoPushTrueResponse =  getAutoPushResponse(resAllFactors, 'push', 'OKTA', true);
+
     var setupSecurityQuestion = _.partial(setup, resAllFactors, { factorType: 'question' });
     var setupGoogleTOTP = _.partial(setup, resAllFactors, { factorType: 'token:software:totp', provider: 'GOOGLE' });
+    var setupGoogleTOTPAutoPushTrue = _.partial(setup, autoPushTrueResponse,
```

Comment:
we could avoid a var here and just do
```
 var setupGoogleTOTPAutoPushTrue = _.partial(setup, getAutoPushResponse(resAllFactors, true),
```
The reason I mention this is because the name of the var ```autoPushTrueResponse``` was confusing to me. We could also change that var to something like ```resAllFactorsAutoPushTrue```, but I think avoiding it might be better